src/components/Onboarding/ChartChooser.tsx

```diff
@@ -40,17 +40,21 @@ const suggestedChartCss = (theme: BifrostTheme) => css`
   .chart-col {
     position: relative;
     max-height: 100%;
+    width: 100%;
     align-self: flex-start;
+    overflow: hidden;
   }
 
   .suggested-charts {
     height: 300px;
```

Comment:
There is still a vertical scroll on the suggested charts when the columns collapse to rows. To fix this, increase the height a bit.
```suggestion
    height: 315px;
```

Other than that looks great!